CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## v1.5.1
+> Chang log since v1.5.0
+
+- Add minReplicas&maxReplicas limitations for subsets of UnitedDeployment ([#1428](https://github.com/openkruise/kruise/pull/1428), [@veophi](https://github.com/veophi))
+
+- Add controller-revision-hash label for ImageListPullJob to control ImagePullJob version ([#1441](https://github.com/openkruise/kruise/pull/1428), [@veophi](https://github.com/veophi))
```

Comment:
consider reorganize the sentence as:

Fix unexpected job recreation by adding  controller-revision-hash label for ImageListPullJob